src/Context/auth-context.js

```diff
@@ -0,0 +1,18 @@
+import { createContext, useContext, useState } from "react";
+const AuthContext = createContext();
+
+const useAuth = () => useContext(AuthContext);
+const token = localStorage.getItem("token");
+
+const AuthProvider = ({ children }) => {
+  const [authState, setAuthState] = useState({
+    isUserLoggedIn: token ? true : false,
+  });
```

Comment:
you can also set `user` object here to show in profile view

Reply:
Its deployed, you can check deploy preview link in netlify robot section